### PR TITLE
Add the "Related resources" section to the already existing CRs

### PR DESCRIPTION
**Pull request by klaudiagrz**
**Description**

Changes proposed in this pull request:

- Add the "Related resources" section to the already existing CRs
- Remove the question mark from the "Mandatory" field in the table

**Related issue(s)**
#1090

### CODE REVIEW

File: docs/application-connector/docs/041-cr-environment-mapping.md [hunk 1]
```diff
@@ -26,7 +26,22 @@ metadata:
...
+
+| Custom resource   |   Description |
+|:----------:|:------|
+| RemoteEnvironment |  Uses this CR to expose RemoteEnvironment's services in a given Environment. |
+
+These components use this CR:
+
+| Component   |   Description |
+|:----------:|:------|
+| Remote Environment Broker |  Uses this CR to enable the provisioning of the ServiceClasses from the given RemoteEnvironment. |
```
Comment (piotrmiskiewicz) [inline docs/application-connector/docs/041-cr-environment-mapping.md]: ... in the environment (to be discussed)

File: docs/application-connector/docs/041-cr-environment-mapping.md [hunk 2]
```diff
@@ -26,7 +26,22 @@ metadata:
...
+| ui-api-layer | Exposes the given CR to the Console UI. |
```
Comment (piotrmiskiewicz) [inline docs/application-connector/docs/041-cr-environment-mapping.md]: Used to filter remote environments enabled in the environment, creates/removes such CR instance to enable/disable (to be discussed)

File: docs/application-connector/docs/041-cr-environment-mapping.md [hunk 3]
```diff
@@ -26,7 +26,22 @@ metadata:
...
+| Custom resource   |   Description |
+|:----------:|:------|
+| RemoteEnvironment |  Uses this CR to expose RemoteEnvironment's services in a given Environment. |
+
+These components use this CR:
+
+| Component   |   Description |
+|:----------:|:------|
+| Remote Environment Broker |  Uses this CR to enable the provisioning of the ServiceClasses in a given Environment. |
+| ui-api-layer | Uses this CR to filter enabled RemoteEnvironments. It also allows to create or delete EnvironmentMappings. |
```
Comment (kazydek) [inline docs/application-connector/docs/041-cr-environment-mapping.md]: allows to -> `allows you to`  (allow sb/sth to do sth)

File: docs/service-brokers/docs/040-cr-event-activation.md [hunk 4]
```diff
@@ -29,9 +29,26 @@ spec:
...
 | **spec.sourceId** |    **YES**   | Used to construct a Publish-Subscribe (Pub/Sub) topic name where the Events are send and from where the Events are consumed. |
+
+## Related resources and components
+
+These are the resources related to this CR:
+
+| Custom resource   |   Description |
+|:----------:|:------|
+| RemoteEnvironment |  Describes a service from which the user receives Events. |
+| Subscription | Contains information on to how create an infrastructure for consuming Events. Works only if the EventActivation is enabled.  |
```
Comment (kazydek) [inline docs/service-brokers/docs/040-cr-event-activation.md]: on to how create -> `on how to create`

File: docs/service-catalog/docs/040-cr-service-binding-usage.md [hunk 5]
```diff
@@ -58,3 +58,21 @@ This table lists all the possible parameters of a given resource together with t
...
+|:----------:|:------|
+| UsageKind |  Provides information where to inject Secrets. |
+| [ServiceBinding](https://kubernetes.io/docs/concepts/extend-kubernetes/service-catalog/#api-resources) |  Provides Secrets to inject.  |
+
+
+These components use this CR:
+
+| Component   |   Description |
+|:----------:|:------|
+| Binding Usage Controller |  Reacts to every action of creating, updating or deleting ServiceBindingUsages in all Namespaces and uses ServiceBindingUsage data to inject binding. |
```
Comment (kazydek) [inline docs/service-catalog/docs/040-cr-service-binding-usage.md]: - creating, updating or deleting -> `creating, updating, or deleting`
- Namespaces and uses -> `Namespaces. It uses` or `Namespaces, and uses`

File: docs/service-catalog/docs/040-cr-service-binding-usage.md [hunk 6]
```diff
@@ -58,3 +58,21 @@ This table lists all the possible parameters of a given resource together with t
...
+| ui-api-layer |  Exposes the given CR to the Console UI. It also allows to create and delete a ServiceBindingUsage. |
```
Comment (kazydek) [inline docs/service-catalog/docs/040-cr-service-binding-usage.md]: allows to create -> `allows you to create` (allow sb/sth to do sth)

File: docs/application-connector/docs/041-cr-environment-mapping.md [hunk 7]
```diff
@@ -26,7 +26,22 @@ metadata:
...
+
+| Custom resource   |   Description |
+|:----------:|:------|
+| RemoteEnvironment |  Uses this CR to expose RemoteEnvironment's services in a given Environment. |
+
+These components use this CR:
+
+| Component   |   Description |
+|:----------:|:------|
+| Remote Environment Broker |  Uses this CR to enable the provisioning of the ServiceClasses in a given Environment. |
```
Comment (kazydek) [inline docs/application-connector/docs/041-cr-environment-mapping.md]: provisioning of the ServiceClasses -> `provisioning of ServiceClasses`

File: docs/service-brokers/docs/040-cr-event-activation.md (hunk 4 above)
Comment (klaudiagrz) [reply docs/service-brokers/docs/040-cr-event-activation.md]: 🙀

File: docs/application-connector/docs/041-cr-environment-mapping.md (hunk 7 above)
Comment (klaudiagrz) [reply docs/application-connector/docs/041-cr-environment-mapping.md]: Oh, my affection for "the + plurals" 😽

File: docs/service-brokers/docs/040-cr-event-activation.md [hunk 8]
```diff
@@ -29,9 +29,26 @@ spec:
...
+These are the resources related to this CR:
+
+| Custom resource   |   Description |
+|:----------:|:------|
+| RemoteEnvironment |  Describes a service from which the user receives Events. |
+| Subscription | Contains information on how to create an infrastructure for consuming Events. Works only if the EventActivation is enabled.  |
+
+These components use this CR:
+
+| Component   |   Description |
```
Comment (PK85) [inline docs/service-brokers/docs/040-cr-event-activation.md]: What's about serverless components, please double check.

Comment (klaudiagrz) [reply docs/service-brokers/docs/040-cr-event-activation.md]: Issue resolved by this PR:
https://github.com/kyma-project/kyma/pull/1221

**Pull request merged**